src/Compilers/Core/Portable/CodeGen/ILBuilderConversions.cs

```diff
@@ -306,6 +309,9 @@ public void EmitNumericConversion(Microsoft.Cci.PrimitiveTypeCode fromPredefType
                             case Microsoft.Cci.PrimitiveTypeCode.Int64:
                                 this.EmitOpCode(ILOpCode.Conv_ovf_u);
                                 break;
+                            case Microsoft.Cci.PrimitiveTypeCode.IntPtr:
```

Comment:
>Microsoft.Cci.PrimitiveTypeCode.IntPtr [](start = 33, length = 38)

Since ```Pointer``` and ```FunctionPointer``` are considered unsigned, it feels like conversion from ```IntPtr``` cannot be a NOP.